<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qrcodepage</class>
 <widget class="QWidget" name="qrcodepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Iosevka Fixed</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="QRcodeImg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>300</width>
     <height>300</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Iosevka Fixed Slab</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>                     QRcode</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>166</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Iosevka Fixed Slab</family>
    </font>
   </property>
   <layout class="QHBoxLayout" name="QRcodeSizeLayout">
    <item>
     <widget class="QSpinBox" name="QRcodeSize">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>40</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="QRcodeSizeText">
      <property name="font">
       <font>
        <family>Iosevka Fixed Slab</family>
       </font>
      </property>
      <property name="text">
       <string>Size 21*21</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>601</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Iosevka Fixed Slab</family>
    </font>
   </property>
   <layout class="QVBoxLayout" name="StringInputLayout">
    <item>
     <widget class="QLabel" name="StringInputText">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Iosevka Fixed Slab</family>
       </font>
      </property>
      <property name="text">
       <string>Input</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="StringInput">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>26</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Iosevka Fixed Slab</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="GenerateButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExportButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>520</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>226</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Iosevka Fixed Slab</family>
    </font>
   </property>
   <layout class="QHBoxLayout" name="ErrorCorrectionLevelLayout">
    <item>
     <widget class="QSpinBox" name="ErrorCorrectionLevel">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ErrorCorrectionLevelText">
      <property name="font">
       <font>
        <family>Iosevka Fixed Slab</family>
       </font>
      </property>
      <property name="text">
       <string>Error Correctiong Level</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="RealSizeTextLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>RealSize:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="RealSizeNumberText">
      <property name="text">
       <string>None</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
